pyanaconda/modules/common/errors/payload.py

```diff
@@ -0,0 +1,28 @@
+#
+# DBus errors related to the payload.
+#
+# Copyright (C) 2019  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from pyanaconda.dbus.error import dbus_error
+from pyanaconda.modules.common.constants.namespaces import PAYLOAD_NAMESPACE
+from pyanaconda.modules.common.errors import AnacondaError
+
+
+@dbus_error("PayloadSourceSetupError", namespace=PAYLOAD_NAMESPACE)
+class PayloadSourceSetupError(AnacondaError):
```

Comment:
I would remove `Payload` from both names of the error. The class will be imported from `payload.py`, so it is clear that it will be related to payload, and the DBus error uses the payload namespace, so `Payload` is already part of its name.

Reply:
Fixed.